### PR TITLE
Reverting the logging to S3 changes

### PR DESCRIPTION
**Pull request by arjunrn**
Temporarily revert the S3 changes so that the PR #543 can be rolled out.

### CODE REVIEW

File: build/setup.d/01-add-node_exporter-text_collector-dir.sh [hunk 1]
```diff
@@ -2,5 +2,4 @@
 
 mkdir -p -m0755 /var/local/textfile_collector
 
-echo "fluentd_process_running{tag=\"td-agent\",hostname=\"$(hostname)\"} 0.0" > /var/local/textfile_collector/fluentd_metrics.prom
-echo "fluentd_default_s3_logging{tag=\"td-agent\",hostname=\"$(hostname)\"} 0.0" > /var/local/textfile_collector/fluentd_default_s3.prom
+echo "fluentd{tag="fluentd_enabled"} 0.0" > /var/local/textfile_collector/fluentd_metrics.prom
```
Comment (szuecs) [inline build/setup.d/01-add-node_exporter-text_collector-dir.sh]: please escape correctly

File: runtime/opt/taupage/init.d/02-register-td-agent.py [hunk 2]
```diff
@@ -47,22 +47,18 @@ def get_scalyr_api_key():
                 raise SystemExit()
         if scalyr_api_key == "Invalid KMS key.":
             logger.error('Failed to decrypt KMS Key')
-            raise SystemExit(1)
+            raise SystemExit()
```
Comment (szuecs) [inline runtime/opt/taupage/init.d/02-register-td-agent.py]: why you change that?
 != 0 is an error and default might be return 0..

File: runtime/opt/taupage/init.d/02-register-td-agent.py [hunk 3]
```diff
@@ -47,22 +47,18 @@ def get_scalyr_api_key():
...
         return scalyr_api_key
 
 
 def update_configuration_from_template():
     ''' Update Jinja Template to create configuration file for Scalyr '''
     fluentd_destinations = dict(scalyr=False, s3=False, rsyslog=False, scalyr_s3=False)
     config = get_config()
-    logging_config = config.get('logging', {})
+    logging_config = config.get('logging')
```
Comment (szuecs) [inline runtime/opt/taupage/init.d/02-register-td-agent.py]: This will crash in line 75 if unset

**Comment by aermakov-zalando** [comment]
I agree with @szuecs's comments; it's unfortunate that we've missed this in the first place. We could keep these fixed, or do you want to keep it as a pure revert PR?

**Comment by christianberg** [comment]
Thanks for the review. Let's keep the fixes @szuecs mentioned in. Would be silly to risk an NPE just to cleanly revert the PR.

**Comment by arjunrn** [comment]
@aermakov-zalando @christianberg @szuecs I've added those changes back. Can you have a look again?

**Comment by aermakov-zalando** [comment]
👍

**Comment by arjunrn** [comment]
:+1:

**Pull request merged**